test/step.test.js

```diff
@@ -0,0 +1,22 @@
+const assert = require('assert');
+const math = require('mathjs');
+
+const stepThroughExpression = require('../lib/simplifyExpression/stepThrough');
+
+function testStepsInExpression(exprStr, outputStr) {
+  it ('test the steps in simplifying expression', function () {
+    assert.equal((stepThroughExpression(math.parse(exprStr))[0]['changeType']), outputStr);
```

Comment:
test the whole node for equivalence to `math.parse('2(x+y)')` like in flatten, I think

lemme know how that works